.github/workflows/pull-request-outbound-proxy.yml

```diff
@@ -0,0 +1,104 @@
+name: Pull Request CI - Outbound Proxy Infra
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch.
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+    paths:
+      - 'infra-l2-outbound-proxy/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Required for sam-validate step
+permissions:
+  id-token: write
+  contents: read
+
+env: # Only adding the variables in that are required for 
+  AWS_REGION: eu-west-2
+
+jobs:
+  run-code-check:
+    name: Code Checks
+    runs-on: ubuntu-latest
+    environment: di-ipv-cri-dev
```

Comment:
we are not deploying in common env any more. We have also removed that from CIC. Not if required for outbound proxy.

Reply:
removed, thanks